src/data/services/LmsApiService.ts

```diff
@@ -462,10 +462,19 @@ class LmsApiService {
     return linkedEnterprisesCopy;
   }
 
-  static fetchEnterpriseLearnerData = async (options) => {
+  static fetchEnterpriseLearnerData = async (enterpriseCustomer?: string, userId?: string, username?: string) => {
```

Comment:
nit: By keeping the arguments inside a single object, we don't have to change any code that was formerly using this.  Also, by putting the arguments in a type it can give us more options for reuse.

```suggestion
  type FetchEnterpriseLearnerDataArgs = {enterpriseCustomer?: string, userId?: string, username?: string};
  static fetchEnterpriseLearnerData = async ({enterpriseCustomer, userId, username}:FetchEnterpriseLearnerDataArgs) => {
```